content/cloud-docs/workspaces/creating.mdx

```diff
@@ -34,48 +31,49 @@ New workspaces can be created by teams with permission to manage workspaces. ([M
+1. (Version control only) Choose a repository from the filterable list. The **Configure settings** page appears.
 
-1. [Navigate to the workspace list](/cloud-docs/workspaces/#listing-and-filtering-workspaces) and click the "+ New Workspace" button (near the top of the page).
+  -> **Note:** Depending on your provider, you may need to change which account's repositories are shown. If the repository is not listed, scroll to the bottom of the list and enter its ID in the text field.
 
-1. On the first screen, choose your VCS provider (or choose "No VCS connection").
+1. Enter a **Workspace Name**. This defaults to the repository name, if applicable. The name must be unique within the organization and can include letters, numbers, dashes (`-`), and underscores (`_`). Refer to our [workspace naming recommendations](/cloud-docs/workspaces/naming).
+1. Add an optional **Description** that will appear at the top of the workspace in the Terraform Cloud UI.
 
-   -> **Note:** If you haven't added a VCS provider for your organization yet, choosing one here will prompt you to configure it. See [Connecting VCS Providers](/cloud-docs/vcs) for more information.
+1. (Version control only) Open **Advanced options** to optionally configure the following additional settings:
+   - [Terraform Working Directory][workdir]
+   - [Automatic Run Triggering][trigger]
+   - [VCS branch][branch]
+   - [Include submodules on clone][submodules]
 
-1. On the second screen, choose a repository from the filterable list. This screen is skipped if you chose "No VCS connection".
+1. Click **Create workspace**.
 
-   Some VCS providers limit the list's size. If a repository isn't listed, you can still choose it by name; scroll to the bottom of the list and enter its ID in the text field.
+For CLI and API-driven workspaces, the workspace overview page appears. For version control workspaces, the **Configure Terraform variables** page appears.
 
-   -> **Note:** For some VCS providers, this list includes a drop-down menu for changing which account's repositories are shown. Other providers combine all available accounts into a single list.
+### Configure Terraform Variables (Version Control Only)
 
-1. On the third screen, enter a name for the workspace. This defaults to the repository name, if applicable. The name must be unique within the organization, and can include letters, numbers, dashes (`-`), and underscores (`_`). See also our [advice for useful workspace names](/cloud-docs/workspaces/naming).
+After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for undefined [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) without default values and automatically creates those variables within the workspace. Terraform cannot perform successful runs until you set values for these undefined variables.
```

Comment:
I not sure about the use of "undefined" here, but need to ponder what it would be instead. Did you find undefined used else where? Maybe Terraform variables with undefined values (i.e. it does not have a default value or value set in the terraform config).

Reply:
Laura came up with this terminology 😄 What makes you unsure about using "undefined" here?

Reply:
good flag! I think we can probably just say "without default values" and remove "undefined". If it doesn't feel right as someone who works with the product, I respect that!

Reply:
I  can see reading undefined as not present in the terraform config -- but i could be over reading this too

Reply:
should we say "After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) without default values and automatically creates those variables within the workspace. Terraform cannot perform successful runs until you set default values for these variables."

@ker-an @rberecka

Reply:
Will update!

Reply:
For the last sentence, we can probably remove the "default":

"Terraform cannot perform successful runs until you set values for these variables."

But aside from that, it makes sense! @rberecka can you have a look? You know more about the variables then I do 🙈

Reply:
@laurapacilio I would remove the word default from the last part of the last sentence and then yes 👍

Reply:
I will update!